src/FeatureItem.svelte

```diff
@@ -5,7 +5,7 @@
 
   export let selected = false;
 
-  export let showPlaceType: false | "always" | "ifNeeded";
+  export let showPlaceType: "no" | "always" | "if-needed";
```

Comment:
uhm, stupid little thing but how do you feel about "never" as opposite for "always"?